src/domain/balances/balances.repository.interface.ts

```diff
@@ -14,6 +16,17 @@ export interface IBalancesRepository {
     excludeSpam?: boolean;
   }): Promise<Balance[]>;
 
+  /**
+   * Gets the collection of {@link Balance} associated with {@link safeAddress}
```

Comment:
```
Gets the collection of {@link SimpleBalance}
```

Reply:
Fixed in [d4cf6711](https://github.com/safe-global/safe-client-gateway/commit/d4cf6711d2b6f399acd83fd4be5b8ff10e1b778d)